### PR TITLE
CORD: Update `generate-node-key` documentation

### LINKED ISSUE #206: Update the README file
https://github.com/dhiway/cord/tree/develop/extras#generate-node-key

We must add an updated version of this information to the main readme file. The directory is outdated. Looks safe to remove.

### PR DESCRIPTION
**Pull request by roynrishingha**
Using:
- `/tmp/cord` directory to mount.
- `dhiway/cord` image as this is our release image.
- `Sparknet` network to connect with.

This PR fixes #206

### CODE REVIEW

**Comment by roynrishingha** [comment]
@amarts @smohan-dw awaiting for review.

File: extras/README.md [hunk 1]
```diff
@@ -7,24 +7,26 @@ This document assumes you have `docker` installed.
...
-docker pull dhiway/cord:develop
+docker pull dhiway/cord
 ```
 
-## Generate `node-key`
+## Generate `node.key`
 
 In a blockchain, every participating node should be uniquely identified. Hence, one needs to generate a new key for the node, which needn't be shared with anyone.
+```sh
+sudo mkdir /tmp/cord && sudo chown $USER /tmp/cord && sudo chmod 777 /tmp/cord
```
Comment (smohan-dw) [inline extras/README.md:17]: How about simplifying the directory handling by having volume details handled by Docker? This requires changes to the docker image.

Comment (roynrishingha) [reply extras/README.md:17]: In order for docker to mount a volume in local machine, we need a local directory. But if we want to use container filesystem itself and not to mount at all, then upon removing the container, we'll loose data. But here we need the node key.

Comment (NiranjanAP) [reply extras/README.md:17]: @roynrishingha  checked with Satish - this comment is not a blocker, I'll approve this change if you have tested this  to be working fine.
For handling directory, lets open another task and discuss on a good way to do it..

**Review by NiranjanAP — APPROVED** [review]

**Pull request merged**